eng/common/tools.ps1

```diff
@@ -436,7 +442,7 @@ function MSBuild() {
   $cmdArgs = "$($buildTool.Command) /m /nologo /clp:Summary /v:$verbosity /nr:$nodeReuse"
 
   if ($warnAsError) { 
-    $cmdArgs += " /p:TreatWarningsAsErrors=true" 
+    $cmdArgs += " /warnaserror /p:TreatWarningsAsErrors=true" 
```

Comment:
Unrelated?

Reply:
Unrelated to the signing issue, but this PR just takes the Arcade's version of tools.ps1.